docs/release-notes/release-notes-0.19.0.md

```diff
@@ -135,6 +135,9 @@
   block. Check
   [here](https://github.com/lightningnetwork/lnd/blob/master/chainio/README.md)
   to learn more.
+  
+* [Decrease the budget for ougtoing HTLCs and
```

Comment:
this could be confusing for users - think we wanna mention that we've removed the 2x output value approach and now use the configured values from `--sweeper.budget.deadlinehtlcratio` and `--sweeper.budget.deadlinehtlc`.